packages/js-dpp/lib/errors/consensus/basic/dataContract/DataContractImmutablePropertiesUpdateError.js

```diff
@@ -0,0 +1,43 @@
+const AbstractBasicError = require('../AbstractBasicError');
+
+class DataContractImmutablePropertiesUpdateError extends AbstractBasicError {
+  /**
+   * @param {{ op: string, path: string }} removedField
```

Comment:
What is the point to pass op if you have already two arguments `removed` and `updated`. Maybe you just make two arguments: `operation` and `path` so logic will be simpler?